sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/StreamingQueryListenerBus.scala

```diff
@@ -39,12 +39,7 @@ class StreamingQueryListenerBus(sparkListenerBus: LiveListenerBus)
    * be dispatched to all StreamingQueryListener in the thread of the Spark listener bus.
    */
   def post(event: StreamingQueryListener.Event) {
-    event match {
```

Comment:
actually I didn't see any reason to distinguish these two types of event when posting